apps/test/unit/templates/SchoolDataInputsTest.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {render, screen, fireEvent} from '@testing-library/react';
+import {expect} from '../../util/deprecatedChai';
+import SchoolDataInputs from '@cdo/apps/templates/SchoolDataInputs';
+import i18n from '@cdo/locale';
+
+describe('SchoolDataInputs', () => {
+  function renderDefault(propOverrides = {}) {
+    render(<SchoolDataInputs {...propOverrides} />);
+  }
+  it('displays headers in basic component render', () => {
```

Comment:
tiny nit: can you add a new line above this test?

Reply:
Done! Thanks!